modules/cli/src/main/scala/scala/cli/commands/Fmt.scala

```
@@ -15,8 +16,64 @@ object Fmt extends ScalaCommand[FmtOptions] {
     List("format"),
     List("scalafmt")
   )
+
+  private def getGitRoot(workspace: os.Path, logger: Logger): Option[String] =
+    try {
+      val result = os.proc("git", "rev-parse", "--show-toplevel").call(cwd = workspace).out.trim
+      Option(result)
+    }
+    catch {
+      case NonFatal(e) =>
```

I think, that we should ignore any exception from above process and use default scalafmt versions. Add only logging what happened.

---

The catch block here is only used for logging. I have adjusted the phrasing to make this come across better - it no longer says "Failed" to not confuse users.